scripts/babel-relay-plugin/src/GraphQLPrinter.js

```diff
@@ -290,6 +290,14 @@ function printFieldsAndFragments(
         fragments.push(printQueryFragment(selection, options));
       } else if (selection.kind === kinds.FIELD) {
         fields.push(selection);
+
+        if (
+          isConnectionType(type) &&
```

Comment:
I can do that if you think this is a reasonable direction for the fix. BTW, in this context, what is the `!!` for?